server/web/errors/index.js

```diff
@@ -15,7 +15,7 @@ exports.register = function (server, options, next) {
       return reply.view('errors/' + statusCode).code(statusCode);
     } else {
       debug(request.response);
-      return reply.view('errors/general');
+      return reply.view('errors/general').code(500);
```

Comment:
our general error wasn't actually reporting as being unhealthy so even if mysql wasn't reachable we were reporting to consul that it was still healthy causing the node to stay in nginx's rotation. now, it'll report a 500 so consul will take it out of rotation and nginx will send 502 if none available.